app/services/authorizer.rb

```diff
@@ -12,8 +12,9 @@ def can?(permission, subject = nil)
       user.permissions.where(:name => permission).present?
     else
       return true if user.admin?
-      collection = @cache[subject.class.to_s][permission] ||= find_collection(subject.class, :permission => permission)
-      collection.include?(subject)
+      @cache["#{subject.class.to_s}-#{permission}-#{subject.id}"] ||=
```

Comment:
@ares if you know this well - let me know here and I'll update it with a comment (or even a new PR would help)